src/module/store/index.ts

```diff
@@ -1,10 +1,27 @@
 import { Node } from 'slate';
 
+export interface ISaveUser {
+  userId: string;
+  organizationId: string;
+}
+
 export const save = (content: Node[]) => {
   const contentString = JSON.stringify(content);
   localStorage.setItem('composeEditor.content', contentString);
 };
 
+export const saveUser = (userObj: ISaveUser) => {
+  const contentString = JSON.stringify(userObj);
+  localStorage.setItem('user', contentString);
```

Comment:
It would be cool if we could structure our datastore and namespace keys. Otherwise the future will likely be harder.

Reply:
Yup - totally agree.

This is a hack for the current iteration. We definitely need an entirely different solution if we continue to use localStorage.

Ideally we have user authentication /authorization so frontend only keeps track of a token or something that is then authenticated.